static/_redirects

```diff
@@ -50,8 +50,9 @@
 /calico/latest/security/encrypt-cluster-pod-traffic /calico/latest/network-policy/encrypt-cluster-pod-traffic 301
 
 # Redirects for CC pages
-/calico-cloud/image-assurance/scan-image-registries#start-the-cli-scanner /calico-cloud/image-assurance/scanners/pipeline-scanner#start-the-scanner 301
-/calico-cloud/image-assurance/scan-image-registries /calico-cloud/image-assurance/scanners/pipeline-scanner 301
+/calico-cloud/image-assurance/scan-image-registries#start-the-cli-scanner /calico-cloud/latest/image-assurance/scanners/pipeline-scanner#start-the-scanner 301
```

Comment:
Redirect rules won't work with #page-refs. These need to be changed to include only the main path.

Browsers will automatically apply page refs at the redirected location. I don't know of a mechanism that can transform the page reference itself.